cms/djangoapps/contentstore/management/commands/tests/test_export_olx.py

```diff
@@ -89,6 +89,7 @@ def test_export_course(self, store_type):
         with tarfile.open(filename) as tar_file:
             self.check_export_file(tar_file, test_course_key)
 
+    @unittest.skip("Bug in django 1.11 prevents this from working in python3.  Re-enable after django 2.x upgrade.")
```

Comment:
Which bug?

Reply:
I put more details in the commit message.  I just updated this to have the same comment here as well.